defaults/main.yml

```diff
@@ -16,3 +17,5 @@ ntp_restricts:
   - "restrict -6 default kod notrap nomodify nopeer noquery"
   - "restrict -6 ::1 nomodify"
 
+auto_ntpdate: no
```

Comment:
Perhaps add a comment here explaining what this does? Better still, a note in the readme file.. a great feature for the role, be good for people to know about it :)